From f88ef2c38bb341534a69bf53d1e043ef71319a8d Mon Sep 17 00:00:00 2001
From: Thomas Genty <tomlohave@geexbox.org>
Date: Tue, 4 Jul 2023 15:41:57 +0200
Subject: [PATCH] hwdb : add support for Archos 101 Cesium Educ to
 60-sensor.hwdb

(cherry picked from commit 41f34dcf3bc7004aaa9b251480d56bcfe144346d)

Related: RHEL-5950
---
 hwdb.d/60-sensor.hwdb | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index 7d1d603daf..d330a05629 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -126,6 +126,10 @@ sensor:modalias:acpi:MXC6655*:dmi:*:svnAquarius*:pnNS483:*
 sensor:modalias:acpi:SMO8500*:dmi:*:svnARCHOS:pnARCHOS80Cesium:*
  ACCEL_MOUNT_MATRIX=-1, 0, 0; 0, 1, 0; 0, 0, 1
 
+# Archos 101 Cesium Educ
+sensor:modalias:acpi:KIOX000A*:dmi:*svn*ARCHOS:*pn*ARCHOS101CesiumEduc*
+ ACCEL_MOUNT_MATRIX=1, 0, 0; 0, -1, 0; 0, 0, 1
+
 #########################################
 # AsusTek
 #########################################
